react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,24 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (args.length === 0) {
+    return undefined;
+  }
+  const items = isArray(args[0]) ? args[0] : args;
+  return Math.min(...items);
```

Comment:
Do you specifically need to return undefined? I would change the order of the if and return only `Math.min(...items);`
By default if you don't return anything it will be undefined. I think you don't need `return undefined`